linux_os/guide/system/network/network-iptables/iptables_ruleset_modifications/ip6tables_rules_for_open_ports/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: ubuntu2204
+
+title: 'Ensure ip6tables firewall rules exist for all open ports'
+
+description: |-
+    Any ports that have been opened on non-loopback addresses
+    need firewall rules to govern traffic.
+
+rationale: |-
+    Without a firewall rule configured for open ports default
+    firewall policy will drop all packets to these ports.
+
+severity: medium
+
+references:
+    cis@ubuntu2204: 3.5.3.2.4
+
+warnings:
```

Comment:
`warnings` should go to the end of the file in this case.
Reference: https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#rule-sections